moped-editor/src/views/dashboard/DashboardView.js

```diff
@@ -100,19 +106,154 @@ const TABS = [
   },
 ];
 
+const UserGreeting = ({ classes, userName }) => {
+  const date = new Date();
+
+  /** Build custom user greeting
+   */
+  return (
+    <Grid className={classes.greeting}>
+      <Typography className={classes.greetingText}>
+        <strong>{`Good ${getTimeOfDay(date)}, ${userName}!`}</strong>
+      </Typography>
+      <Typography variant="h1" className={classes.date}>
+        {getCalendarDate(date)}
+      </Typography>
+    </Grid>
+  );
+};
+
+/** Hook that provides memoized column settings */
+const useColumns = ({ refetch, handleSnackbar, classes }) =>
+  useMemo(() => {
+    return [
+      {
+        headerName: "ID",
+        field: "project_id",
+        editable: false,
+        renderCell: ({ row }) => (
+          <div className={classes.tableRowDiv}>{row.project_id}</div>
+        ),
+        flex: 0.5,
+      },
+      {
+        headerName: "Full name",
+        field: "project_name_full",
+        editable: false,
+        valueGetter: (value) => value.trim(),
+        renderCell: ({ row }) => (
+          <div className={classes.tableRowDiv}>
+            <Link
+              component={RouterLink}
+              to={`/moped/projects/${row.project_id}`}
+              sx={{
+                overflow: "hidden",
+                textOverflow: "ellipsis",
+                display: "block",
+                fontSize: 16,
+              }}
+            >
+              {row.project_name_full}
+            </Link>
+          </div>
+        ),
+        flex: 4,
+      },
+      {
+        headerName: "Status",
+        field: "moped_proj_phases",
+        editable: false,
+        valueGetter: (value) => value[0]?.moped_phase.phase_name,
```

Comment:
"Unknown" technically isnt sorting alphabetically correctly here, this is how i handled this in a diff pr i worked on migrating the subprojects table to datagrid pro!  https://github.com/cityofaustin/atd-moped/commit/175e779ddef40ae6546fb760a22d88b076b20094

Reply:
thanks for catching that!! i've implemented your fix :)